src/config/index.ts

```diff
@@ -397,12 +391,11 @@ if (acgConfig) {
     }
 
     // Kafka settings
-    const broker = clowdAppConfig.kafka.brokers[0];
+    const kafka = clowdAppConfig.kafka;
+    const broker = kafka.brokers[0];
 
-    data.kafka = {
-        host: broker.hostname,
-        port: broker.port.toString()
-    };
+    // create array of host:port pairs from config for multi-broker support
+    data.kafka.brokers = _.map(kafka.brokers, broker => `${broker.hostname}:${broker.port}`);
```

Comment:
Variable `broker` shadowing.

I'd suggest to move the `const broker` closer to where it is first used.

Reply:
Fair point - I'll just rename the variable used in the closure.